### ISSUE TITLE
Energy Puller Module & Reverse Energy Distributor

### ISSUE THREAD

**Issue by thepacksmith** [desht/ModularRouters#222]
### Describe the feature

I would like to have energy puller modules and have a toggle to reverse energy distributor modules.

### Reasons why it should be considered

This, for me, is the only thing that this mod is missing and it doesn't make sense not to include it seeing as it's the only function it can't offer.

**Comment by desht** [comment]
What's a use-case for energy puller modules? It's easy to pipe energy into a router via pretty much any mod which handles energy.

Will consider a direction toggle for the distributor.  Guess it makes sense, given the item distributor can do it.

**Comment by thepacksmith** [comment]
The use-case is the same as for the item modules. It allows for a cable less setup. You could say the same about the need for an item puller module, since you could pipe items around right? 

An example use-case would be with the Powah mod and Flux Networks. If I was to have 8 thermo gens and wanted to pull the power into flux plugs the result is a very ugly cable spaghetti, or a lot of expensive flux plugs. With an energy distributor set to pull mode it would pull the power from all 8 generators and could then output that power into a flux plug.

There are many more use-cases for this module.

**Comment by desht** [comment]
I'm asking about the use-case for an energy puller module, not a direction toggle for the distributor.

To be clear: I'm ok with the idea of adding a pull mode for the distributor.  I'm not sure I see the need for a separate puller module to be added.

**Comment by thepacksmith** [comment]
Ah I see.

The only real use case for an energy puller would be for a single generator setup if you wanted it to be free of cables.

**Comment by desht** [comment]
Yeah, not enough of a use case :)  Especially since generators as a rule actively push their energy to neighbours.

But I'll keep this open for the distributor direction toggle.
